target-bfin/bfin-sim.c

```diff
@@ -525,7 +525,7 @@ decode_PushPopReg_0(DisasContext *dc, uint16_t iw0)
     int W   = ((iw0 >> PushPopReg_W_bits) & PushPopReg_W_mask);
     int grp = ((iw0 >> PushPopReg_grp_bits) & PushPopReg_grp_mask);
     int reg = ((iw0 >> PushPopReg_reg_bits) & PushPopReg_reg_mask);
-    TCGv treg, tmp;
+    TCGv treg = 0, tmp;
```

Comment:
this func is weird.  i don't think this is the right change, but i'll have to look more to figure it out.